pkg/runtime/runtime_test.go

```diff
@@ -441,6 +443,8 @@ func TestInitPubSub(t *testing.T) {
 			assert.Nil(t, err)
 		}
 
+		rt.startSubscribing()
```

Comment:
Is this enough to validate this scenario? What about `startReadingFromBinding` ? Can you please add more test for this scenario?

Reply:
`startReadingFromBinding` is already tested in runtime_test.go `TestReadInputBindings` method